Content.Server/Administration/Systems/AdminVerbSystem.Smites.cs

```diff
@@ -846,5 +848,31 @@ private void AddSmiteVerbs(GetVerbsEvent<Verb> args)
             Message = Loc.GetString("admin-smite-terminate-description")
         };
         args.Verbs.Add(terminate);
+
+        Verb superslip = new()
+        {
+            Text = "Super Slip",
```

Comment:
Needs localisation.

Reply:
so you want me to add localisation for every single admin smite's text?

Reply:
> I assume just this one unless you want to go through and do all the others while adding it here

sure why not